crates/common/types/genesis.rs

```diff
@@ -371,6 +378,69 @@ impl Genesis {
         });
         Trie::compute_hash_from_unsorted_iter(iter)
     }
+    pub fn write_as_json(&self, path: &Path) -> Result<(), String> {
+        let genesis_json = serde_json::to_string(&self)
+            .map_err(|e| format!("Could not convert genesis to string: {}", e))?;
+        let genesis_as_map: Map<String, Value> = serde_json::from_str(&genesis_json)
+            .map_err(|e| format!("Failed to de-serialize genesis file: {}", e))?;
+        // Keys sorted based off this ethpandaops example:
+        // https://github.com/ethpandaops/ethereum-genesis-generator/blob/master/apps/el-gen/mainnet/genesis.json
```

Comment:
Can we also order the config fields internally as in the example? That is, 

```
- "chainId",
- "homesteadBlock"
- "daoForkBlock"
- "daoForkSupport"
- "eip150Block"
- "eip150Hash"
- "eip155Block"
- "eip158Block"
- "byzantiumBlock"
- "constantinopleBlock"
- "petersburgBlock"
- "istanbulBlock"
- "muirGlacierBlock"
- "berlinBlock"
- "londonBlock"
- "arrowGlacierBlock"
- "grayGlacierBlock"
- "terminalTotalDifficulty"
- "shanghaiTime"
- "cancunTime"
- "pragueTime"
- "ethash"
- "depositContractAddress"
- "blobSchedule"
```

This fields also have a reasonable order in them (forks are ordered, blob schedule is last)

Reply:
Sure!

Reply:
Done